Makefile

```diff
@@ -16,7 +16,7 @@ SHARE = Event.pm Snap.pm parse_cm_file.pl arc_time_machine.pl \
 	timeline.js timeline.css vert_line.gif \
 	task_schedule.cfg
 
-DATA =	ACE_hourly_avg.npy
+DATA =
```

Comment:
What's this about? Seems unrelated.

Reply:
So in general I'm left with "do I use my commit messages to store history or just copy them all into the PR text"?  Here, the Makefile doesn't work in a clean repo because the npy is missing, so I stuck that in ee23dbc.  If we want it in a separate PR that's fine.  If we want to update the Makefile so that the process to build a new npy file is run before trying to install it that's fine.

Reply:
These kinds of things go into the PR description. I don't mind unrelated changes in a PR like this when we are just trying to get stuff done, but you need to document the changes and rationale in the PR description. If I saw such a description I would have just said "fine" to myself and moved on, though it raises the question of whether a clean install without that file actually works. I don't remember if that file gets auto-created or if some process will just fail because of a missing file.